assembl/static2/js/app/components/administration/survey/configureThematicForm.jsx

```diff
@@ -33,25 +37,42 @@ export function getFieldName(themeId: string, values: ThemesValue, fieldName: st
   return result;
 }
 
-const ConfigureThematicForm = ({ editLocale, values, thematicId }: Props) => {
-  const name = getFieldName(thematicId, values ? values.themes : [], 'themes');
-  return (
-    <div className="form-container">
-      <Field
-        required
-        editLocale={editLocale}
-        name={`${name}.title`}
-        component={MultilingualTextFieldAdapter}
-        label={`${I18n.t('administration.tableOfThematics.thematicTitle')} ${editLocale.toUpperCase()}`}
-      />
-      <Field
-        deleteTooltip={deleteThematicImageTooltip}
-        name={`${name}.img`}
-        component={FileUploaderFieldAdapter}
-        label={I18n.t('administration.tableOfThematics.headerLabel')}
-      />
-    </div>
-  );
-};
+class ConfigureThematicForm extends React.PureComponent<Props> {
+  constructor(props: Props) {
+    super(props);
+    const { values, thematicId, slug } = this.props;
```

Comment:
since you have access to 'props' as the constructor's argument here isn't it better to use instead of this.props ? No clue if this makes any performance difference or difference at all

Reply:
no difference...